### PR TITLE
Template readme

### PR DESCRIPTION
**Pull request by luotigerlsx**
I have finished the draft for the readme without CI/CD session. Please take a look.

### CODE REVIEW

File: sklearn/sklearn-template/template/README.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+Cloud ML Engine (CMLE) at scale. Before you jump in, let’s cover some of the different tools you’ll be using to get 
+your job running on CMLE.
+
+- [Google Cloud Platform](https://cloud.google.com/) lets you build and host applications and websites, store data, 
+and analyze data on Google's scalable infrastructure.
+- [Cloud Machine Learning Engine](https://cloud.google.com/ml-engine/) is a managed service that enables you to easily build 
+machine learning models that work on any type of data, of any size.
+- [Google Cloud Storage](https://cloud.google.com/storage/) (GCS) is a unified object storage for developers and 
+enterprises, from live data serving to data analytics/ML to data archiving.
+- [Cloud SDK](https://cloud.google.com/sdk/) is a command line tool which allows you to interact with
```
Comment (cfezequiel) [inline sklearn/sklearn-template/template/README.md]: We should probably mention that this SDK provides CLI tools such as the `gcloud` command line tool for interacting with Google Cloud products and services.

File: sklearn/sklearn-template/template/README.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+NUMERIC_FEATURES = [
+    'sepal_length',
+    'sepal_width',
+    'petal_length',
+    'petal_width',
+]
+
+CATEGORICAL_FEATURES = [
+
```
Comment (cfezequiel) [inline sklearn/sklearn-template/template/README.md:73]: Since we are using Iris, this doesn't need to be filled. But I think we need to give a comment to fill this with any categorical features in the dataset.

File: sklearn/sklearn-template/template/README.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+- Configure project id as environment variable
+  ```bash
+  $ export PROJECT_ID=[your-google-project-id]
+  ```
+  
+## Step 1. Modify metadata.py
+```python
+# Example for iris dataset
+COLUMNS = None  # Schema of the data. Necessary for data stored in GCS
```
Comment (cfezequiel) [inline sklearn/sklearn-template/template/README.md]: I didn't think we'd need another `COLUMNS` variable. Couldn't we just re-use `FEATURE_COLUMNS`?

File: sklearn/sklearn-template/template/README.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+for iris dataset (can be found at `bigquery-public-data.ml_datasets.iris`). In most cases, only the following
+items need to be modified, in order to adapt to the target dataset. 
+- **COLUMNS**: the schema of ths data, only required for data stored in GCS
+- **NUMERIC_FEATURES**: columns those will be treated as numerical features
+- **CATEGORICAL_FEATURES**: columns those will be treated as categorical features
+- **LABEL**: column that will be treated as label
+
+## Step 2. Submit ML training job
+```shell
+bash scripts/train.sh [INPUT_PATH] [RUN_ENV] [BUCKET_NAME]
```
Comment (cfezequiel) [inline sklearn/sklearn-template/template/README.md]: Regarding BUCKET_NAME, I'm inclined to omit this from the script params as it doesn't really change much during model dev (refer to my comment in PR#9 for train.sh).

File: sklearn/sklearn-template/template/README.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+```
+- INPUT_PATH: Dataset to use for training and evaluation. Can be BigQuery table or a file (CSV).
+              BigQuery table should be specified as `PROJECT_ID.DATASET.TABLE_NAME`.
+- RUN_ENV: Whether to run `local` (on-prem) or `remote` (GCP).
+- BUCKET_NAME: GSC bucket to be used for storing the trained model and evaluation result.
+
+## Step 3. Deploy the trained scikit-learn model
+After training finishes, the model will be exported to specified job directory in Google Cloud Storage.
```
Comment (cfezequiel) [inline sklearn/sklearn-template/template/README.md:123]: Nonblock:
I was thinking of adding a script for deploying the model. With that, we can potentially do the CD part of CI/CD.

**Review by cfezequiel — CHANGES_REQUESTED** [review]
Thanks for preparing the readme. Added some comments.

File: sklearn/sklearn-template/template/README.md (hunk 1 above)
Comment (luotigerlsx) [reply sklearn/sklearn-template/template/README.md]: Modified.

File: sklearn/sklearn-template/template/README.md (hunk 2 above)
Comment (luotigerlsx) [reply sklearn/sklearn-template/template/README.md:73]: Added.

File: sklearn/sklearn-template/template/README.md (hunk 3 above)
Comment (luotigerlsx) [reply sklearn/sklearn-template/template/README.md]: This is mainly for data in GCS and we assume there is no header in the data. Since the raw data schema may have more columns then needed, we cannot simply reuse feature columns.

File: sklearn/sklearn-template/template/README.md (hunk 4 above)
Comment (luotigerlsx) [reply sklearn/sklearn-template/template/README.md]: Modified.

**Comment by luotigerlsx** [comment]
I have modified accordingly. Please take a look.

File: sklearn/sklearn-template/template/README.md (hunk 3 above)
Comment (cfezequiel) [reply sklearn/sklearn-template/template/README.md]: I see. Would suggest renaming to `CSV_COLUMNS` so it's less ambiguous and indicate that this is for the case where the CSV data in GCS doesn't have a header.

**Comment by cfezequiel** [comment]
Added just one comment.

File: sklearn/sklearn-template/template/README.md (hunk 3 above)
Comment (luotigerlsx) [reply sklearn/sklearn-template/template/README.md]: Modified.
